index.html

```diff
@@ -31,12 +31,25 @@ <h1><a href="https://midwestopensourcealliance.org/">MOSA</a></h1>
 
         <h2 id="welcome-to-midwest-open-source-alliance">Welcome to Midwest Open Source Alliance</h2>
 
-        <p>Midwest Open Source Alliance promotes the education, support and development of Drupal and related open source software within the Midwest Drupal community. We are an Illinois nonprofit and <a href="https://www.charitynavigator.org/index.cfm?bay=search.profile&ein=831560285">federal 501(c)(3)</a> tax exempt organization.</p>
-
-        <p>We are fiscal sponsor for open source software events in the Midwest, including <a href="https://www.midcamp.org/">Midwest Area Drupal Camp (MidCamp)</a>, <a href="http://tcdrupal.org/">Twin Cities Drupal Camp</a>, and MWDS, as well as the <a href="https://www.meetup.com/drupalchicago/">Drupal Chicago Meetup group</a>.</p>
-
-        <p>In addition, we are helping support initiatives that improve the event experience for both speakers and attendees, especially in areas that promote inclusivity and accessibility. We are currently supporting the <a href="https://opencollective.com/drupal-recording-initiative">Drupal Recording Initiative</a> and working to help launch the <a href="https://github.com/MidCamp/live-captioning">Live Captioning Initiative</a>.</p>
-
+        <p>Midwest Open Source Alliance promotes the education, support and development of Drupal and related open source software within the Midwest Drupal community (and beyond). We are an Illinois nonprofit and were granted <a href="https://www.charitynavigator.org/index.cfm?bay=search.profile&ein=831560285">federal 501(c)(3)</a> tax exempt organization status in 2018 (EIN 83-1560285).</p>
+
+        <p>We are fiscal sponsor for:
+            
+        <ul>
+            <li><a href="https://www.midcamp.org/">Midwest Area Drupal Camp (MidCamp)</a> - founding member</li>
+            <li><a href="http://tcdrupal.org/">Twin Cities Drupal Camp</a> - founding member</li>
+            <li>Midwest Drupal Summit - founding member</li>
+            <li><a href="https://www.meetup.com/drupalchicago/">Drupal Chicago Meetup group</a></li>
```

Comment:
This feels out of place in a list of member organizations, so I'd say to drop it into the paragraph below

Reply:
```suggestion
```